releases/v1.1.md

```diff
@@ -0,0 +1,73 @@
+    <a href="https://binaries.cockroachdb.com/cockroach-v1.1.0.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+## Summary
+
+This section summarizes the most significant, user-facing changes included in v1.1. For a complete list of changes, including bug fixes and performance improvements, see the [release notes](index.html/#testing-releases) for previous testing releases.
+
+- [SQL Statements](#sql-statements)
+- [SQL Types](#sql-types)
+- [Cockroach Commands](#cockroach-commands)
+- [Admin UI](#admin-ui)
+- [Known Limitations](#known-limitations)
+
+### SQL Statements
+
+Statement | Description
+----------|------------
+[`IMPORT TABLE`](../v1.1/import-table.html) | Use this new statement to import an entire table's data into a running cluster by loading CSV files.
+[`SHOW QUERIES`](../v1.1/show-queries.html)<br>[`CANCEL QUERY`](../v1.1/cancel-query.html) | Use these new statements to monitor the progress of active queries, and, if necesssry, cancel long-running queries causing unwanted resource consumption.
+[`SHOW SESSIONS`](../v1.1/show-sessions.html) | Use this new statement to monitor the overall state of client connections and identify those that may need further investigation or adjustment.
+[`SHOW JOBS`](show-jobs.html) | Use this new statement to monitor the progress of schema changes and enterprise backup and restores.
+[`CANCEL JOB`](../v1.1/cancel-job.html)<br>[`PAUSE JOB`](../v1.1/pause-job.html)<br>[`RESUME JOB`](../v1.1/resume-job.html) | Use these new statements to cancel or pause and then resume enterprise backups and restores.
+[`SHOW BACKUPS`](../v1.1/show-backups.html) | Use this new statement to list the contents of an enterprise backup.
+[`DROP USER`](../v1.1/drop-user.html) | Use this new statement to remove one or more SQL users. This features is equivalent to the existing [`cockroach user rm`](../v1.1/create-and-manage-users.html) command.
+[`DROP DATABASE`](../v1.1/drop-database.html) | Use the new `CASCADE` option to drop all tables and views in a database as well as all objects (such as [constraints](constraints.html) and [views](views.html)) that depend on those tables. This option does not list objects it drops, so should be used cautiously.
+[`INSERT`](../v1.1/insert.html#update-values-on-conflict) | Use a `WHERE` clause with `ON CONFLICT` to conditionally apply updates that conflict a Unique constraint on one or more columns.
+
+### SQL Types
+
+Type | Description
+-----|------------
+[`ARRAY`](../v1.1/array.html) | Use this new data type to store 1-dimensional, 1-indexed, homogenous arrays of any non-array data type.
+[`UUIC`](../v1.1/uuid.html) | Use this new data type to store 128-bit values that are globally unique. This type is recommended for [auto-generating unique row IDs](../v1.1/uuid.html#create-a-table-with-auto-generated-unique-row-ids).
```

Comment:
Should this be [`UUID`](..) instead of [`UUIC`](..)?

Reply:
Yep, good catch. Thanks, @jtescher.